include/LLVMSPIRVLibC.h

```diff
@@ -0,0 +1,139 @@
+//===- LLVMSPIRVLibC.h - Read and write SPIR-V binary -------------*- C -*-===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+// Copyright (c) 2020 Julia Computing. All rights reserved.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a
+// copy of this software and associated documentation files (the "Software"),
+// to deal with the Software without restriction, including without limitation
+// the rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Software, and to permit persons to whom the
+// Software is furnished to do so, subject to the following conditions:
+//
+// Redistributions of source code must retain the above copyright notice,
+// this list of conditions and the following disclaimers.
+// Redistributions in binary form must reproduce the above copyright notice,
+// this list of conditions and the following disclaimers in the documentation
+// and/or other materials provided with the distribution.
+// Neither the names of Advanced Micro Devices, Inc., nor the names of its
```

Comment:
This license header looks a bit odd.  As this PR adds a number of new files, I wonder if it makes sense to add them under the "new" Khronos Apache 2.0 license instead?  See https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/80#issuecomment-476781085 .

Reply:
Sure, I just copied over from neighboring files in my editor :slightly_smiling_face: I'll have a look at that issue.